### ISSUE TITLE
Topic in latest ArXiV documents at 2022-03-01 22:54:06

### ISSUE THREAD

**Issue by wanted2** [wanted2/caineng.in#527]
**Is your feature request related to a problem? Please describe.**
Topic in latest ArXiV documents at 2022-03-01 22:54:06
**Describe the solution you'd like**
* Topic 0: 
  * _Learning Explicit and Implicit Latent Common Spaces for Audio-Visual  Cross-Modal Retrieval_
  * _Efficient Single Image Super-Resolution Using Dual Path Connections with  Multiple Scale Learning_
  * _Learning Deep Semantic Model for Code Search using CodeSearchNet Corpus_
  * _Learning Multi-Object Dynamics with Compositional Neural Radiance Fields_
  * _A Novel Anomaly Detection Method for Multimodal WSN Data Flow via a  Dynamic Graph Neural Network_
  * _A Novel Framework for Brain Tumor Detection Based on Convolutional  Variational Generative Models_
  * _BLINC: Lightweight Bimodal Learning for Low-Complexity VVC Intra Coding_
  * _Asymmetric Hash Code Learning for Remote Sensing Image Retrieval_
  * _Bridging the Gap Between Patient-specific and Patient-independent  Seizure Prediction via Knowledge Distillation_
  * _Reinforcement Routing on Proximity Graph for Efficient Recommendation_
* Topic 1: 
  * _Audio Visual Scene-Aware Dialog Generation with Transformer-based Video  Representations_
  * _Reproducibility in Optimization: Theoretical Framework and Limits_
  * _Minimax in Geodesic Metric Spaces: Sion's Theorem and Algorithms_
  * _Long-Time Convergence and Propagation of Chaos for Nonlinear MCMC_
  * _Single Trajectory Nonparametric Learning of Nonlinear Dynamics_
  * _A PDE-Based Analysis of the Symmetric Two-Armed Bernoulli Bandit_
  * _Differentially Private Regression with Unbounded Covariates_
  * _Finding Safe Zones of policies Markov Decision Processes_
  * _Generalized Bayesian Additive Regression Trees Models: Beyond  Conditional Conjugacy_
* Topic 2: 
  * _Graph Spring Network and Informative Anchor Selection for Session-based  Recommendation_
  * _Learning Multi-granularity User Intent Unit for Session-based  Recommendation_
  * _Hierarchical User Intent Graph Network forMultimedia Recommendation_
  * _Intention Adaptive Graph Neural Network for Category-aware Session-based  Recommendation_
  * _Multi-view Intent Disentangle Graph Networks for Bundle Recommendation_
  * _Disentangled Graph Neural Networks for Session-based Recommendation_
  * _The Unfairness of Active Users and Popularity Bias in Point-of-Interest  Recommendation_
  * _A Systematic Analysis on the Impact of Contextual Information on  Point-of-Interest Recommendation_
* Topic 3: 
  * _Unsupervised Representation Learning for Point Clouds: A Survey_
  * _RIConv++: Effective Rotation Invariant Convolutions for 3D Point Clouds  Deep Learning_
  * _A QoE Model in Point Cloud Video Streaming_
  * _Patch-Based Deep Autoencoder for Point Cloud Geometry Compression_
  * _Compressed Geometric Arrays for Point Cloud Processing_
  * _ANTLER: Bayesian Nonlinear Tensor Learning and Modeler for Unstructured,  Varying-Size Point Cloud Data_
  * _Point Set Self-Embedding_
  * _PCSCNet: Fast 3D Semantic Segmentation of LiDAR Point Cloud for  Autonomous Car using Point Convolution and Sparse Convolution Network_
* Topic 4: 
  * _Learned Video Compression with Residual Prediction and Loop Filter_
  * _Soundify: Matching Sound Effects to Video_
  * _MANet: Improving Video Denoising with a Multi-Alignment Network_
  * _FoV Privacy-aware VR Streaming_
  * _Punctuation Restoration_
  * _Improving Content-Aware Video Streaming in Congested Networks with  In-Network Computing_
  * _GenéLive! Generating Rhythm Actions in Love Live!_
  * _DCNGAN: A Deformable Convolutional-Based GAN with QP Adaptation for  Perceptual Quality Enhancement of Compressed Video_
  * _Nebula: Reliable Low-latency Video Transmission for Mobile Cloud Gaming_
* Topic 5: 
  * _repro_eval: A Python Interface to Reproducibility Measures of  System-oriented IR Experiments_
  * _Grep-BiasIR: A Dataset for Investigating Gender Representation-Bias in  Information Retrieval Results_
  * _Atomized Search Length: Beyond User Models_
  * _HC4: A New Suite of Test Collections for Ad Hoc CLIR_
  * _Probably Reasonable Search in eDiscovery_
  * _Googling for Abortion: Search Engine Mediation of Abortion Accessibility  in the United States_
  * _Similarity Search on Computational Notebooks_
  * _Customising Ranking Models for Enterprise Search on Bilingual  Click-Through Dataset_
  * _Search Strategy Formulation for Systematic Reviews: issues, challenges  and opportunities_
  * _Audio Retrieval with Natural Language Queries: A Benchmark Study_
* Topic 6: 
  * _Confidence Calibration for Object Detection and Segmentation_
  * _Defect detection and segmentation in X-Ray images of magnesium alloy  castings using the Detectron2 framework_
  * _Emotion Recognition from Multiple Modalities: Fundamentals and  Methodologies_
  * _Music Sentiment Transfer_
  * _Predicting emotion from music videos: exploring the relative  contribution of visual and auditory information to affective responses_
  * _Tailor Versatile Multi-modal Learning for Multi-label Emotion  Recognition_
  * _Video Background Music Generation with Controllable Music Transformer_
  * _A Novel Multi-Task Learning Method for Symbolic Music Emotion  Recognition_
* Topic 7: 
  * _ODSearch: A Fast and Resource Efficient On-device Information Retrieval  for Mobile and Wearable Devices_
  * _Similarity search on neighbor's graphs with automatic Pareto optimal  performance and minimum expected quality setups based on hyperparameter  optimization_
  * _Query Suggestion for Click-Absent Queries in Enterprise Search_
  * _An Efficient Pruning Process with Locality Aware Exploration and Dynamic  Graph Editing for Subgraph Matching_
  * _(2.5+1)D Spatio-Temporal Scene Graphs for Video Question Answering_
  * _Tree-based Search Graph for Approximate Nearest Neighbor Search_
* Topic 8: 
  * _Musical Audio Similarity with Self-supervised Convolutional Neural  Networks_
  * _Automated Reasoning in Non-classical Logics in the TPTP World_
* Topic 9: 
  * _Deep Learning, Natural Language Processing, and Explainable Artificial  Intelligence in the Biomedical Domain_
  * _Using word clouds for fast identification of papers' subject domain and  reviewers' competences_
  * _Relational Algebra and Calculus with SQL Null Values_
  * _DoCoGen: Domain Counterfactual Generation for Low Resource Domain  Adaptation_
  * _Statistical Relational Artificial Intelligence with Relative  Frequencies: A Contribution to Modelling and Transfer Learning across Domain  Sizes_
  * _DGSS : Domain Generalized Semantic Segmentation using Iterative Style  Mining and Latent Representation Alignment_
  * _Priming Cross-Session Motor Imagery Classification with A Universal Deep  Domain Adaptation Framework_
  * _Cost-effective Framework for Gradual Domain Adaptation with  Multifidelity_
* Topic 10: 
  * _Fostering the Robustness of White-Box Deep Neural Network Watermarks by  Neuron Alignment_
* Topic 11: 
  * _Centre Symmetric Quadruple Pattern: A Novel Descriptor for Facial Image  Recognition and Retrieval_
  * _Local Gradient Hexa Pattern: A Descriptor for Face Recognition and  Retrieval_
  * _Cascaded Asymmetric Local Pattern: A Novel Descriptor for Unconstrained  Facial Image Recognition and Retrieval_
  * _Local Quadruple Pattern: A Novel Descriptor for Facial Image Recognition  and Retrieval_
  * _R-Theta Local Neighborhood Pattern for Unconstrained Facial Image  Recognition and Retrieval_
  * _Local Directional Gradient Pattern: A Local Descriptor for Face  Recognition_
  * _Malakai: Music That Adapts to the Shape of Emotions_
  * _Symbolic Music Loop Generation with VQ-VAE_
  * _MusIAC: An extensible generative framework for Music Infilling  Applications with multi-level Control_
  * _Diversity in the Music Listening Experience: Insights from Focus Group  Interviews_
* Topic 12: 
  * _Robust Reinforcement Learning as a Stackelberg Game via  Adaptively-Regularized Adversarial Training_
  * _Gapoera: Application Programming Interface for AI Environment of  Indonesian Board Game_
  * _Avalanche RL: a Continual Reinforcement Learning Library_
  * _Sound Adversarial Audio-Visual Navigation_
  * _Attacks and Faults Injection in Self-Driving Agents on the Carla  Simulator -- Experience Report_
  * _Selection, Ignorability and Challenges With Causal Fairness_
  * _Diffusion Causal Models for Counterfactual Estimation_
  * _Deep Causal Reasoning for Recommendations_
  * _The Causal Marginal Polytope for Bounding Treatment Effects_
* Topic 13: 
  * _Perceptual Consistency in Video Segmentation_
  * _ciscNet -- A Single-Branch Cell Instance Segmentation and Classification  Network_
  * _Seamless Copy Move Manipulation in Digital Images_
  * _SketchEdit: Mask-Free Local Image Manipulation with Partial Sketches_
* Topic 14: 
  * _Multi-Cue Adaptive Emotion Recognition Network_
* Topic 15: 
  * _Recent Advances and Challenges in Deep Audio-Visual Correlation Learning_
  * _Transformers in Medical Image Analysis: A Review_
  * _Adversarial Example Devastation and Detection on Speech Recognition  System by Adding Random Noise_
  * _3DRM:Pair-wise relation module for 3D object detection_
  * _DL-SLOT: Dynamic Lidar SLAM and Object Tracking Based On Graph  Optimization_
* Topic 16: 
  * _Understanding and Improving Graph Injection Attack by Promoting  Unnoticeability_
  * _SEC4SR: A Security Analysis Platform for Speaker Recognition_
  * _Unsupervised Graph Poisoning Attack via Contrastive Loss  Back-propagation_
* Topic 17: 
  * _Optimal Clustering with Bandit Feedback_
  * _Use Image Clustering to Facilitate Technology Assisted Review_
  * _Relational Causal Models with Cycles:Representation and Reasoning_
  * _Learning Causal Overhypotheses through Exploration in Children and  Computational Models_
* Topic 18: 
  * _Clustering by Hill-Climbing: Consistency Results_
  * _A sampling-based approach for efficient clustering in large datasets_
  * _Using the left Gram matrix to cluster high dimensional data_
  * _Strong Consistency for a Class of Adaptive Clustering Procedures_
  * _Mind Your Clever Neighbours: Unsupervised Person Re-identification via  Adaptive Clustering Relationship Modeling_
  * _Clustering by the Probability Distributions from Extreme Value Theory_
  * _Human Motion Detection Using Sharpened Dimensionality Reduction and  Clustering_
  * _Self-Evolutionary Clustering_
* Topic 19: 
  * _Exploring with Sticky Mittens: Reinforcement Learning with Expert  Interventions via Option Templates_
* Topic 20: 
  * _Rule-based Evolutionary Bayesian Learning_
  * _Large-scale Stochastic Optimization of NDCG Surrogates for Deep Learning  with Provable Convergence_
  * _A Robust Multi-Objective Bayesian Optimization Framework Considering  Input Uncertainty_
* Topic 21: 
  * _Unsupervised physics-informed disentanglement of multimodal data for  high-throughput scientific discovery_
  * _Improving Fake News Detection by Using an Entity-enhanced Framework to  Fuse Diverse Multimodal Clues_
* Topic 22: 
  * _Tackling benign nonconvexity with smoothing and stochastic gradients_
  * _The Power of Adaptivity in SGD: Self-Tuning Step Sizes with Unbounded  Gradients and Affine Variance_
  * _Support Vectors and Gradient Dynamics for Implicit Bias in ReLU Networks_
  * _Interacting Contour Stochastic Gradient Langevin Dynamics_
  * _Meta-Learning for Simple Regret Minimization_
* Topic 23: 
  * _Multimedia Datasets for Anomaly Detection: A Survey_
  * _Provable Stochastic Optimization for Global Contrastive Learning: Small  Batch Does Not Harm Performance_
* Topic 24: 
  * _Robust Training under Label Noise by Over-parameterization_
  * _The Role of Reusable and Single-Use Side Information in Private  Information Retrieval_
* Topic 25: 
  * _Bidding Agent Design in the LinkedIn Ad Marketplace_
* Topic 26: 
  * _FakeAVCeleb: A Novel Audio-Video Multimodal Deepfake Dataset_
  * _Deepfake Detection for Facial Images with Facemasks_
  * _Model Attribution of Face-swap Deepfake Videos_
  * _A Survey on Patients Privacy Protection with Stganography and Visual  Encryption_
* Topic 27: 
* Topic 28: 
  * _Multiscale Crowd Counting and Localization By Multitask Point  Supervision_
  * _Wisdom of (Binned) Crowds: A Bayesian Stratification Paradigm for Crowd  Counting_
  * _FusionCount: Efficient Crowd Counting via Multiscale Feature Fusion_
  * _An End-to-End Transformer Model for Crowd Localization_
* Topic 29: 
* Topic 30: 
  * _Reinforcing Local Feature Representation for Weakly-Supervised Dense  Crowd Counting_
* Topic 31: 
  * _Speaker Identity Preservation in Dysarthric Speech Reconstruction by  Adversarial Speaker Adaptation_
  * _Phase Continuity: Learning Derivatives of Phase Spectrum for Speech  Enhancement_
  * _Benchmarking Generative Latent Variable Models for Speech_
  * _Speaker Adaptation Using Spectro-Temporal Deep Features for Dysarthric  and Elderly Speech Recognition_
* Topic 32: 
* Topic 33: 
  * _Deconstructing Distributions: A Pointwise Framework of Learning_
* Topic 34: 
  * _Integrating psychotherapy practices and gamified elements in novel game  mechanics for stress relief_
  * _M2I: From Factored Marginal Trajectory Prediction to Interactive  Prediction_
  * _Efficient and Differentiable Conformal Prediction with General Function  Classes_
* Topic 35: 
  * _Fighting Game Commentator with Pitch and Loudness Adjustment Utilizing  Highlight Cues_
  * _Sobolev Transport: A Scalable Metric for Probability Measures with Graph  Metrics_
  * _How player and opponent personalities influence cooperative gameplay_
  * _Learning to Identify Perceptual Bugs in 3D Video Games_
  * _Large-scale Personalized Video Game Recommendation via Social-aware  Contextualized Graph Neural Network_
* Topic 36: 
  * _FUNQUE: Fusion of Unified Quality Evaluators_
* Topic 37: 
* Topic 38: 
  * _Seeing Like a Toolkit: How Toolkits Envision the Work of AI Ethics_
* Topic 39: 
* Topic 40: 
* Topic 41: 
* Topic 42: 
  * _Graph Convolutional Networks for Multi-modality Medical Imaging:  Methods, Architectures, and Clinical Applications_
  * _Application of DatasetGAN in medical imaging: preliminary studies_
* Topic 43: 
  * _Nonlinear Discrete Optimisation of Reversible Steganographic Coding_
* Topic 44: 
  * _Inferring Lexicographically-Ordered Rewards from Preferences_
* Topic 45: 
  * _Decision support system for distributed manufacturing based on  input-output analysis and economic complexity_
  * _Can Online Customer Reviews Help Design More Sustainable Products? A  Preliminary Study on Amazon Climate Pledge Friendly Products_
  * _Extraction of Product Specifications from the Web -- Going Beyond Tables  and Lists_
* Topic 46: 
  * _A Complete Criterion for Value of Information in Soluble Influence  Diagrams_
  * _On PAC-Bayesian reconstruction guarantees for VAEs_
* Topic 47: 
  * _Robotic Telekinesis: Learning a Robotic Hand Imitator by Watching Humans  on Youtube_
* Topic 48: 
  * _Towards Reducing Manual Workload in Technology-Assisted Reviews:  Estimating Ranking Performance_
  * _OCR-IDL: OCR Annotations for Industry Document Library Dataset_
  * _Understanding and Mitigating the Effect of Outliers in Fair Ranking_
* Topic 49: 

**Describe alternatives you've considered**
None
**Additional context**
None

**Comment by wanted2** [comment]
This issue was created by an aggregate batch
